kafka-ui-react-app/src/redux/reducers/clusters/selectors.ts

```diff
@@ -24,3 +24,9 @@ export const getOnlineClusters = createSelector(getClusterList, (clusters) =>
 export const getOfflineClusters = createSelector(getClusterList, (clusters) =>
   clusters.filter(({ status }) => status === ServerStatus.OFFLINE)
 );
+
+export const getClustersReadonlyStatus = (clusterName: string) =>
+  createSelector(
+    getClusterList,
+    (clusters) => clusters.find(({ name }) => name === clusterName)?.readOnly
```

Comment:
I think it's better to have a more explicit value here. If there is no value, we should return `false`. Any non-boolean value doesn't add any logic, so it's pointless to have `undefined`.